src/main/kotlin/me/ezra_home/retail_software_solution/model/entity/FruitEntityDto.kt

```diff
@@ -0,0 +1,22 @@
+package me.ezra_home.retail_software_solution.model.entity
+
+import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Size
+import java.io.Serializable
+import java.math.BigDecimal
+import java.time.OffsetDateTime
+import java.util.*
+
+/**
+ * DTO for {@link me.ezra_home.retail_software_solution.model.entity.FruitEntity}
+ */
+data class FruitEntityDto(
+    val id: UUID? = null,
+    @field:NotNull @field:Size(max = 255) val name: String? = null,
```

Comment:
You can drop the @field annotations. We will do validations in the service layer with appropriate error messages